### PR TITLE
Added Apache as a guest member

### PR DESCRIPTION
**Pull request by linkfang**
[Issue1164](https://github.com/jakartaee/jakarta.ee/issues/1164)

### CODE REVIEW

File: layouts/shortcodes/pages/membership/explore-members.html
Comment (autumnfound) [inline layouts/shortcodes/pages/membership/explore-members.html]: I'd recommend using width 145 here. While we want to keep them small, we want them to be comparable in size and legible

Comment (linkfang) [reply layouts/shortcodes/pages/membership/explore-members.html]: OK, cool.
I tried 145 and 130, I think 130 might be better. 145 looks a little big compared to others.
What do you think?

130:
image

145:
image

**Review by autumnfound — APPROVED** [review]

**Comment by linkfang** [comment]
Got a +1 from Zahra, good to merge after code review

**Pull request merged**
